### PR TITLE
Fix Address Overview on mobile

### LINKED ISSUE #15: Update Address overview to fit on mobile
As a user, I want the address overview to be responsive, so that I can view it on my mobile. 

# Replication steps 

 1. In mobile view, click ‘Explore’ > ‘Transaction’

 2. Then click an address link in the `From` or To `column`

The address overview page has images overlapping with text and poor formatting. Please update it, to make it match the attached design.

# Prerequisites

You will need to know basic Javascript (jQuery) and css (scss) to do this task. The knowledge of nunjucks will be helpful.

# Guidelines & Hints

For the Transactions section at the end of the page, we always want to show the scrollbar, so the user knows in advance that there is more data to be seen. 

You can check the code used in the Menu -> Transactions page.

The address overview page is located at: views/pages/address-detail.njk.

The current scrollbar is implemented on views/pages/tx-list.njk (look for className scroll-bar-wrapper).

The code to handle the scrollbar is located at: src/assets/js/mobile/scroll-bar.js (you should not have to change this).

Add any other necessary mobile css to _mobile.scss to match the design.

Please make sure your code doesn’t affect the desktop version. 

We do not want to change the HTML structure. 

We also recommend putting all mobile related styling code to the `_mobile.scss` file. 

# Resources

## Repository link

https://github.com/cennznet/explorer/tree/master 

This ‘explorer’ repository contains several projects. 

The project you need to work on is in the ‘portal’ folder.

## Design:

Please refer to the InVision prototype here:  [UNcover Bounty Tasks](https://invis.io/7TUN9MQY259)

You will need to request access permission to access inspect mode on the InVision prototype. 

Screenshot of intended design also attached. 

*Please note there may be some inconsistencies with the InVision prototype and the live CSS. If this occurs, please use the styles defined by the exisiting CSS. 

## Get Up & Running 

Please check the README and CONTRIBUTING files:

https://github.com/cennznet/explorer/blob/master/portal/README.md https://github.com/cennznet/explorer/blob/master/portal/CONTRIBUTING.md 

# Test & Verify Your Changes 

## Mobile Devices 

Ensure your changes work on: 

- The latest Android and iPhones phones.
- Chrome, Safari and FireFox responsive views.

Ensure your changes work as expected and look exactly like the attached design. 

Also ensure that the changes do not introduce bugs or break the desktop version.

## Expected Outcome

**Given** user navigates to address overview page using mobile device

**When** user navigates to address section

**Then** user should see QR code with address

**And**  user should see full address

**And** user should see copy symbol

**Given** user navigates to address overview page using mobile device

**When** user navigates to address section

**And** user click the copy symbol

**Then** user should have address copied to the clipboard and a tooltip appears

**Given** user navigates to address overview page using mobile device

**When** user navigates to assets section

**Then** user should see all assets stored in that address

**Given** user navigates to address overview page using mobile device

**When** user navigates to assets section

**And** user click the buttons on the left top side of the assets section

**Then** user should see the assets displaying style changes

**Given** user navigates to address overview page using mobile device

**When** user navigates to transactions section

**Then** user should see time

 **And** user should see address

 **And** user should see incoming

 **And** user should see outgoing

 **And** user should see type

  **And** user should see status

  **And** user should see balance

**Given** user navigates to address overview page using mobile device

**When** user navigates to transactions section

 **And** user clicks on arrow sign at the end of the line

**Then** user should see expanded section

 **And** user should see block

 **And** user should see transaction

 **And** user should see  size 

 **And** user should see  fee

**Given** user navigates to address overview page using mobile device

**When** user navigates to transactions section

**Then** user should see transaction type selection dropdown

**Given** user navigates to address overview page using mobile device

**When** user navigates to transactions section

**Then** user should see transaction flow selection dropdown

**Given** user navigates to address overview page using mobile device

**When** user navigates to transactions section

**Then** user should see start date selection dropdown

**Given** user navigates to address overview page using mobile device

**When** user navigates to transactions section

**Then** user should see end date selection dropdown

# Current Screenshots

 
image

image

–––––––––––––––––––––––––––––––––––––––––––

# Designed Screen (how your design should look)
image

### PR DESCRIPTION
**Pull request by iamonuwa**
Fixes #15

### CODE REVIEW

File: portal/views/pages/address-detail.njk [hunk 1]
```diff
@@ -7,15 +7,16 @@
 {% block main %}
   <h1 class="main-heading">{{ _('section.addressOverview') }}</h1>
   {% include 'partials/address/address-info.njk' %}
+  <hr />
   {% include 'partials/address/address-tokens.njk' %}
 
-  <div class="block-section">
+  <div class="block-section is-mobile address-detail-page">
```
Comment (AlexZhenWang) [inline portal/views/pages/address-detail.njk]: Hi, just wondering, I see the `is-mobile` class is added to this div, will that make these code 
```
<h2>{{ _('navbar.transactions') }} ({{ total | numberFormat }} {{ _('helper.field.total') }})</h2>
```
 to be not displayed on desktop?

Comment (iamonuwa) [reply portal/views/pages/address-detail.njk]: Remove all duplicates of is-mobile

File: portal/locales/en.json [hunk 2]
```diff
@@ -116,5 +116,27 @@
 	"User-generated": "User-generated",
 	"helper.field.show.mobile": "Show list of validators",
 	"helper.field.hide.mobile": "Hide list of validators",
-	"navbar.filter.transactions": "Filter Transactions"
+	"navbar.filter.transactions": "Filter Transactions",
+	"filter.token": "Token",
+	"filter.selectToken": "Select Token",
+	"95800000000000000000": "95800000000000000000",
```
Comment (AlexZhenWang) [inline portal/locales/en.json]: maybe I ask why do we add these?

File: portal/yarn.lock [hunk 3]
```diff
@@ -26,10 +26,10 @@
     date-fns "^1.27.2"
```
Comment (AlexZhenWang) [inline portal/yarn.lock]: I don't think it is necessary to change yarn.lock file

File: portal/src/assets/style/_mobile.scss [hunk 4]
```diff
@@ -1,545 +1,718 @@
...
-  }
-
-  .mobile-home-view-all-btn {
-    flex: 0.3 !important;
-  }
-
-  .mobile-home-tx-table tr:not(:nth-child(-n+5)) {
-    display: none;
-  }
+	.footer {
```
Comment (AlexZhenWang) [inline portal/src/assets/style/_mobile.scss]: Please do not change the indent here

File: portal/src/assets/style/_mobile.scss [hunk 5]
```diff
@@ -1,545 +1,718 @@
...
-      display: inline;
-      flex: 3;
-      white-space: normal;
-      word-break: break-word;
-      color: #1E2022;
-    }
-  }
+.tx-summary-content,
+.block-section__content {
+	.row {
```
Comment (AlexZhenWang) [inline portal/src/assets/style/_mobile.scss]: Please do not change the indent here, thanks

**Comment by iamonuwa** [comment]
Closing this in favour of #22

**Pull request closed without merging**